src/actors/FilterBarActor.js

```diff
@@ -0,0 +1,88 @@
+import * as SharedUtils from '../utils/SharedUtils';
+
+export class FilterBarActor {
```

Comment:
Events. Otherwise we're creating a two-way data flow system by proxy, and have the inconvenience of passing the Actor right down the component tree so that a button component x layers deep can trigger state changes.

Why should a Button have to know anything about how its used? It's a button, it's job is to be pressed.